Sources/PrimerSDK/Classes/Core/Payment Services/PayPalService.swift

```diff
@@ -57,16 +57,20 @@ internal class PayPalService: PayPalServiceProtocol {
             return completion(.failure(PrimerError.payPalSessionFailed))
         }
 
-        guard let urlScheme = settings.urlScheme else {
-            return completion(.failure(PrimerError.payPalSessionFailed))
+        guard var urlScheme = settings.urlScheme else {
+            return completion(.failure(PrimerError.missingURLScheme))
+        }
+        
+        if urlScheme.suffix(3) == "://" {
+            urlScheme = urlScheme.replacingOccurrences(of: "://", with: "")
         }
 
         let body = PayPalCreateOrderRequest(
             paymentMethodConfigId: configId,
             amount: amount,
             currencyCode: currency,
-            returnUrl: urlScheme,
-            cancelUrl: urlScheme
+            returnUrl: "\(urlScheme)://paypal-success",
```

Comment:
maybe we should refactor some of this into it's own method since code is repeated? Just thinking loud